.github/workflows/release-api-model.yaml

```diff
@@ -0,0 +1,100 @@
+name: release api-model
+on:
+  workflow_dispatch:
+    inputs:
+      release-version:
+        description: 'Version being released'
+        required: true
+      snapshot-version:
+        description: 'Next snapshot version'
+        required: true
+      branch:
+        description: 'Branch to release from'
+        required: true
+        default: 'main'
+jobs:
+  release-api-model:
+    runs-on: ubuntu-18.04
+    if: github.repository_owner == 'Apicurio'
+    steps:
+      - name: Log Metadata
+        run: |
+          echo "==========================================================================================================="
+          echo "Release Module: apicurio-registry-operator-api-model"
+          echo "Release Version: ${{ github.event.inputs.release-version }}"
+          echo "Release Branch: ${{ github.event.inputs.branch }}"
+          echo "Release Actor: ${{ env.GITHUB_ACTOR }}"
+          echo "Next Snapshot Version: ${{ github.event.inputs.snapshot-version }}"
+          echo "==========================================================================================================="
+      
+      - name: Set up JDK 11
+        uses: AdoptOpenJDK/install-jdk@v1
+        with:
+          version: '11'
+          architecture: x64
+
+      - name: Set up settings.xml
+        run: |
+          pwd
+          mkdir -p /home/runner/.m2
+          chmod 755 /home/runner/.m2
+          echo "<settings><servers><server><id>${{ secrets.OSSRH_ID }}</id><username>${{ secrets.OSSRH_USERNAME }}</username><password>${{ secrets.OSSRH_TOKEN }}</password></server></servers><profiles><profile><id>${{ secrets.OSSRH_ID }}</id><activation><activeByDefault>true</activeByDefault></activation><properties><gpg.executable>gpg</gpg.executable><gpg.passphrase>${{ secrets.GPG_PASSPHRASE}}</gpg.passphrase></properties></profile></profiles></settings>" > /home/runner/.m2/settings.xml
+          cat /home/runner/.m2/settings.xml
+
+      - name: Configure Git
+         run: |
+          git config --global user.name "apicurio-ci"
+          git config --global user.email "apicurio.ci@gmail.com"
+          
+       - name: Checkout Code with Branch ${{github.event.inputs.branch}}
+         uses: actions/checkout@v2
+         with:
+           ref: ${{github.event.inputs.branch}}
+           token: ${{ secrets.ACCESS_TOKEN }}
+           
+       - name: list files
+         run: ls -lrt
+      
+      - name: Update Release Version ${{ github.event.inputs.release-version}}
+        run: ./mvnw versions:set -DnewVersion=${{ github.event.inputs.release-version}} -DgenerateBackupPoms=false -DprocessAllModules=true
+
+      - name: Build Project
+        run: ./mvnw clean install
```

Comment:
cd api-model

Please change the directory to api-model in the below steps as well before executing the maven commands